javacord-core/src/main/java/org/javacord/core/util/handler/channel/ChannelUpdateHandler.java

```diff
@@ -163,6 +167,9 @@ private void handleServerChannel(JsonNode jsonChannel) {
                         overwrittenPermissions.put(entity.getId(), newOverwrittenPermissions);
                         dispatchServerChannelChangeOverwrittenPermissionsEvent(
                                 channel, newOverwrittenPermissions, oldOverwrittenPermissions, entity);
+                        areYouAffected.compareAndSet(false,entity instanceof User && ((User) entity).isYourself());
+                        areYouAffected.compareAndSet(false,entity instanceof Role
```

Comment:
I don't want to be nitpicking, but there's missing a space (also in the line above). 😉 
```suggestion
                        areYouAffected.compareAndSet(false, entity instanceof Role
```